app/models/form1095_b.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+class Form1095B < ApplicationRecord
+  has_kms_key
+  encrypts :form_data, key: :kms_key, **lockbox_options
+
+  # validations
+  validates :veteran_icn, :tax_year,  presence: true
+  validates :veteran_icn, uniqueness: { scope: :tax_year }
+  validate :proper_form_data_schema
+
+  # scopes
+  scope :get_available_forms, lambda { |icn|
+    where(veteran_icn: icn).order(tax_year: :desc).pluck(:tax_year, :updated_at)
+  }
+
+  # methods
+  def get_pdf
+    generate_pdf
+  end
+
+  private
+
+  def country_and_zip
+    "#{data['country']} #{data['zip_code'] || data['foreign_zip']}"
+  end
+
+  def middle_initial
+    data['middle_name'] ? data['middle_name'][0] : ''
+  end
+
+  # rubocop:disable Metrics/MethodLength
+  def generate_pdf
+    pdf = PdfForms.new(Settings.binaries.pdftk)
+
+    temp_file = Tempfile.new("1095B-#{SecureRandom.hex}.pdf")
+    pdf_template = "lib/pdf_fill/forms/pdfs/f1095bs/1095b-#{tax_year}.pdf"
+
+    unless File.exist?(pdf_template)
+      Rails.logger.error "1095-B template for year #{tax_year} does not exist."
+      raise "1095-B for tax year #{tax_year} not supported"
+    end
+
+    pdf.fill_form(
+      pdf_template,
+      temp_file,
+      {
+        "topmostSubform[0].Page1[0].Pg1Header[0].cb_1[1]": data['is_corrected'] && 2,
```

Comment:
What are you trying to do here? `true && 2` will return 2 and `false && 2` will return false. Is this the expected behavior?

Reply:
I could not tell you why pdf_fill requires the number 2 in order to check this box on the pdf, but after a while of trying to figure it out, 2 worked. So yes, this is the expected behavior